src/Components/Form/DeleteButton.js

```diff
@@ -11,10 +11,9 @@ export default function DeleteButton({ id }) {
 		<ButtonDelete
 			type="button"
 			onClick={async () => {
-				const response = await fetch('/api/post/' + id, {
+				const _response = await fetch('/api/post/' + id, {
```

Comment:
if you don't want to use the response you can just omit the constant declaration:
```suggestion
				await fetch('/api/post/' + id, {
```

Reply:
Good to know. I would still like to keep it as I was very pleased with my solution.